plugin/src/main/java/io/snyk/languageserver/protocolextension/SnykExtendedLanguageClient.java

```diff
@@ -85,7 +116,8 @@ private void showAuthenticatedMessage() {
     ServerMessageHandler.showMessage("Authentication with Snyk successful", messageParams);
```

Comment:
This may be redundant, as we already send a notification from LS. Please check :)

Reply:
Quick check seems to indicate that it's required (we don't get this notification when it is commented out)